packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -36,7 +36,7 @@ export default class QuanticCategoryFacet extends LightningElement {
     this.facet = CoveoHeadless.buildCategoryFacet(engine, {
       options: {
         field: this.field,
-        delimitingCharacter: ';',
+        delimitingCharacter: '|',
```

Comment:
Why are you changing the facet delimiter here? Is that because of this field format (`atlgeographicalhierarchy`)?

If this specific field is delimited using pipes, then we should add an attribute to the `quantic-category-facet` to be able to override the default delimiter.

Reply:
I think I wrongly assumed we used the incorrect default. But you are probably right, it's probably just the field that is using a different delimiter than the default.

Reply:
Providing an @api attribute to specify the delimiter makes sense too. We'll do it when we redo these facets though.